.github/workflows/publish.yml

```diff
@@ -0,0 +1,53 @@
+name: Publish gem
+
+on: workflow_dispatch
+
+jobs:
+  # TODO: Prepare release notes
+
+  release-gem:
+    name: Build and push gem to RubyGems.org
+    runs-on: ubuntu-24.04
+    environment: "rubygems" # see: https://github.com/DataDog/dd-trace-rb/settings/environments
+    concurrency: "rubygems" # Only one publish job at a time
+
+    permissions:
+      id-token: write
+      contents: write
+
+    outputs:
+      version: ${{ steps.version.outputs.version }}
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.3'
+      - id: version
+        run: |
+          echo "version=$(ruby -e 'puts Gem::Specification::load(Dir.glob("*.gemspec").first).version')" >> $GITHUB_OUTPUT
+      - uses: rubygems/release-gem@v1
+
+  github-release:
```

Comment:
Shouldn't that be `rubygems-release`? I mean, it would be nice to publish it on GitHub Packages for Ruby as well, in which case _that one_ would be called `github-release`.